crypto/keyring/record.go

```diff
@@ -62,9 +79,19 @@ func (k Record) GetAddress() (types.AccAddress, error) {
 	return pk.Address().Bytes(), nil
 }
 
-// TODO consider to remove it
 func (k Record) GetType() KeyType {
```

Comment:
Since `Record.GetType()` method is used in [readTxCommandFlags](https://github.com/cosmos/cosmos-sdk/blob/cyberbono3/keyring-migration/client/cmd.go#L243) and in [GetFromFields](https://github.com/cosmos/cosmos-sdk/blob/cyberbono3/keyring-migration/client/context.go#L350) I decided to modify this method and leave as it is. What do you think, @amaurym ?

Reply:
After discussion with @amaurym, we decided to leave `Record.GetType()`